README.md

```diff
@@ -1,40 +1,48 @@
 
-A module for provisioning an IBM Cloud Red Hat OpenShift cluster on VPC Gen2. The module either creates the required Cloud Object Storage instance or uses an existing instance. The module also supports optionally passing a key management configuration for secret encryption and boot volume encryption
+A module for provisioning an IBM Cloud Red Hat OpenShift cluster on VPC Gen2. The module either creates the required Cloud Object Storage instance or uses an existing instance. The module also supports optionally passing a key management configuration for secret encryption and boot volume encryption.
 
-## Before you begin
+### Before you begin
 
 - Make sure that you have a recent version of the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
 - Make sure that you have a recent version of the [IBM Cloud Kubernetes service CLI](https://cloud.ibm.com/docs/containers?topic=containers-kubernetes-service-cli)
 
-## Usage
+<!-- Below content is automatically populated via pre-commit hook -->
+<!-- BEGIN OVERVIEW HOOK -->
+## Overview
+* [terraform-ibm-base-ocp-vpc](#terraform-ibm-base-ocp-vpc)
+* [Submodules](./modules)
+    * [fscloud](./modules/fscloud)
+* [Examples](./examples)
+    * [2 MZR clusters in same VPC example](./examples/multiple_mzr_clusters)
+    * [Advanced example (mzr, auto-scale, kms, taints)](./examples/advanced)
+    * [Basic single zone example](./examples/basic)
+    * [Cluster security group rules example](./examples/add_rules_to_sg)
+    * [Financial Services compliant example](./examples/fscloud)
+* [Contributing](#contributing)
+<!-- END OVERVIEW HOOK -->
+
+<!-- This heading should always match the name of the root level module (aka the repo name) -->
+## terraform-ibm-base-ocp-vpc
+
+### Usage
 ```hcl
-# Replace "master" with a GIT release version to lock into a specific release
 module "ocp_base" {
-  # update this value to the value of your IBM Cloud API key
-  ibmcloud_api_key     = "ibm cloud api key" # pragma: allowlist secret
-  source  = "terraform-ibm-modules/base-ocp-vpc/ibm"
-  version = "latest" # Replace "latest" with a release version to lock into a specific release
+  ibmcloud_api_key     = "XXXXXXXXXXXXXXXXXXX"
+  source               = "terraform-ibm-modules/base-ocp-vpc/ibm"
+  version              = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
   cluster_name         = "example-cluster-name"
-  # modify the value for resource_group_id with and id of a group you own
-  resource_group_id    = "id of existing resource group"
+  resource_group_id    = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
   region               = "us-south"
   force_delete_storage = true
-  vpc_id               = "id of existing VPC"
-  ## obtain the below values from the targeted VPC and adjust to the number of zones,
-  ## subnets, subnet name, cidr_block, id, zone
+  vpc_id               = "79cxxxx-xxxx-xxxx-xxxx-xxxxxXX8667"
```

Comment:
can we remove digits?

Reply:
why? I did this to show its not a secret value.